pymatgen/io/qchem_io/sets.py

```diff
@@ -37,9 +39,9 @@ def __init__(self, molecule, job_type, basis_set, SCF_algorithm, DFT_rung=4, PCM
             raise ValueError('molecule must be a Pymatgen Molecule object!')
```

Comment:
Pls remove this test. In general, it is a VERY bad idea to do type testing. The whole point of Python is to use duck typing. What if someone supplied something which effectively is like a molecule but not called a molecule?

Reply:
I'll remove it, thanks. FYI @samblau